src/Symfony/Component/HttpClient/CurlHttpClient.php

```diff
@@ -274,6 +274,11 @@ public function request(string $method, string $url, array $options = []): Respo
             $curlopts[CURLOPT_TIMEOUT_MS] = 1000 * $options['max_duration'];
         }
 
+        if (isset($options['extra']['curl']) && \is_array($options['extra']['curl'])) {
+            $curlopts += $options['extra']['curl'];
```

Comment:
i think it's legit to override computed options then? Otherwise we'd be silently ignoring those.

Reply:
I think we should not allow overriding curlopts that are already abstracted by the options from httpclient.
BTW, the title of the PR misleading in this regard.